pkg/accountmanager/poller_test.go

```diff
@@ -0,0 +1,358 @@
+package accountmanager
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	mock "github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"antrea.io/nephe/apis/crd/v1alpha1"
+	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
+	"antrea.io/nephe/pkg/inventory"
+	"antrea.io/nephe/pkg/labels"
+	"antrea.io/nephe/pkg/testing/cloud"
+)
+
+var _ = Describe("Account Poller", func() {
+	Context("Account poller workflow", func() {
+		var (
+			credentials               = "credentials"
+			testAccountNamespacedName = types.NamespacedName{Namespace: "namespace01", Name: "account01"}
+			testSecretNamespacedName  = types.NamespacedName{Namespace: "namespace01", Name: "secret01"}
+			testCesNamespacedName     = types.NamespacedName{Namespace: "namespace01", Name: "Ces01"}
+			account                   *v1alpha1.CloudProviderAccount
+			accountPollerObj          *accountPoller
+			fakeClient                client.WithWatch
+			secret                    *corev1.Secret
+			ces                       *v1alpha1.CloudEntitySelector
+			mockCtrl                  *mock.Controller
+			mockCloudInterface        *cloud.MockCloudInterface
+			pollIntv                  uint
```

Comment:
Reorg the declarations, as mentioned earlier.

Reply:
Solved.